users/pharnb/index.md

```diff
@@ -10,8 +10,8 @@ Ben Pharn
 
 I am a first year Master's student here at Oregon State University, having completed my undergraduate degree here as well.  I was born and raised in Southern California, where the majority of my family still resides.
 
-[Email] <a href="mailto:pharnb@oregonstate.edu" target="top"> pharnb@oregonstate.edu </a> (pharnb@oregonstate.edu)
-[Resume] (./files/resume.pdf)
+[Email]<a href="mailto:pharnb@oregonstate.edu" target="top"> pharnb@oregonstate.edu </a> (pharnb@oregonstate.edu)
+/n [Resume](./files/resume.pdf)
```

Comment:
oops, my example was supposed to say:

first line< space >< space >
second line < space >< space >
third line

An alternative is hitting enter twice and creating a new paragraph.